src/lib.rs

```diff
@@ -47,7 +51,7 @@ impl Repository {
                 destination_must_be_empty: true,
                 ..Default::default()
             },
-            open::Options::isolated(),
+            open::Options::default().permissions(Permissions::all()),
```

Comment:
there is also https://docs.rs/gix/latest/gix/open/struct.Permissions.html#method.secure, but looking at the source it is exactly the same